src/routes/MapFeed/MapFeed.connector.js

```diff
@@ -0,0 +1,100 @@
+import { connect } from 'react-redux'
```

Comment:
i wonder how much we can reuse Feed.connector somehow instead of duplicating so much...

Reply:
Originally I thought MapFeed would be doing a whole lot more but since then I've reconsidered. Want to get closer to actually implementing the whole map view before ripping it out.